src/ex6/client/src/reducers/items-view-reducer.js

```diff
@@ -0,0 +1,23 @@
+import {
+  getUndoneTodos,
+  getSearchedTodos,
+} from "../selectors/items-entities-selectors";
+
+const initialState = { filteredTodos: null };
+
+const itemsViewReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case "FILTER_UNDONE_TODOS":
```

Comment:
same here, use enum for each group of actions so when you use it everywhere in the code you can reuse the same enum and also avoid typo and bugs